src/components/blocks/Endpoint/Endpoint.js

```diff
@@ -19,11 +19,13 @@ function Endpoint(props) {
         />
         <InputText
           data-test-id="EndpointInputText"
-          placeholder="ex: http://localhost:3000/api/users"
+          placeholder={
+            labels.placeholder || 'ex: http://localhost:3000/api/users'
```

Comment:
instead of `OR` condition. we can keep default object and then override given object